website/templates/profile/account.mako

```diff
@@ -61,17 +60,20 @@
                                     <td style="word-break: break-all;">{{ $data.address }}</td>
                                     <td style="width:150px;"><a data-bind="click: $parent.resendConfirmation">resend&nbsp;confirmation</a></td>
                                     <td style="width:50px;" ><a data-bind="click: $parent.removeEmail"><i class="fa fa-times text-danger"></i></a></td>
-                                </tr>
-                                <!-- /ko -->
+                                 </tr>
```

Comment:
Sorry to nit-pick, but would you mind removing this space? That way, the closing tag is indented at the same level as it's corresponding opening tag.